pkg/resolver/searchHelper.go

```diff
@@ -62,6 +62,9 @@ func getWhereClauseExpression(prop, operator string, values []string) []exp.Expr
 		lhsExp = goqu.C(prop)
 	} else {
 		lhsExp = goqu.L(`"data"->>?`, prop)
+		if dataType == "number" {
+			lhsExp = goqu.L(`("data"->>?)?`, prop, goqu.L("::numeric"))
```

Comment:
Should we use `->` instead of `->>` ?

`->>` is equivalent to adding `::text` so it feels like we are adding an unnecessary step.

Reply:
Done